easybuild/easyblocks/f/fdtd_solutions.py

```diff
@@ -40,7 +44,23 @@ def configure_step(self):
 
     def build_step(self):
         """No build step for FDTD Solutions."""
-        pass
+        # locate RPM and rebuild it to make it relocatable
+        rpms = glob.glob(os.path.join(self.cfg['start_dir'], 'rpm_install_files', 'FDTD-%s*.rpm' % self.version))
+        if len(rpms) != 1:
+            self.log.error("Incorrect number of RPMs found, was expecting exactly one: %s" % rpms)
+        rebuilt_dir = os.path.join(self.cfg['start_dir'], 'rebuilt')
+        rebuild_rpm(rpms[0], rebuilt_dir)
+
+        # replace original RPM with relocatable RPM
+        rebuilt_rpms = glob.glob(os.path.join(rebuilt_dir, '*', '*.rpm'))
+        if len(rebuilt_rpms) != 1:
+            self.log.error("Incorrect number of rebuilt RPMs found, was expecting exactly one: %s" % rebuilt_rpms)
+
+        try:
+            os.rename(rpms[0], '%s.bk' % rpms[0])
```

Comment:
mostly for debugging purposes, this will be cleaned up once the installation succeeds anyway